action.yml

```diff
@@ -32,6 +32,17 @@ runs:
           alias log_error="echo \"ERROR:\""
         fi
         set -e
+        
+        mkdir -p ${{ inputs.install-dir }}
+
+        if [[ ${{ inputs.cosign-release }} == "main" ]]; then
+          log_info "installing cosign via 'go install' from its main version"
+          GOBIN=$(go env GOPATH)/bin
+          go install github.com/sigstore/cosign/cmd/cosign@main
+          ln -s $GOBIN/cosign ${{ inputs.install-dir}}/cosign
+          echo "${{ inputs.install-dir }}" >> $GITHUB_PATH
```

Comment:
```suggestion
```
this is done below in a platform-independent way so i think you can remove this line